### PR TITLE
Work correctly with option's aliases

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: options.ts [hunk 1]
```diff
@@ -26,7 +26,16 @@ export class OptionsBase {
 
 		yargs.options(this.options);
 		this.argv = yargs.argv;
-
+		
+		// Check if some of the given options is alias and add its original name to argv
+		_.each(_.keys(this.argv), item => {
+			_.each(this.optionNames, optionName => {
```
Comment (rosen-vladimirov) [inline options.ts]: I think you can use ```_.each(this.optionNames, (optionName, optionValue) => { ```
which will simplify the if below to:
```TypeScript
if(optionValue.alias === item)
```

**Comment by rosen-vladimirov** [comment]
:+1:  Just one idea for improvement, but nothing critical :)

File: options.ts (hunk 1 above)
Comment (Fatme) [inline options.ts]: Actually the `optionValue` is the first parameter :smile:

Comment (rosen-vladimirov) [inline options.ts]: argh, ignore my comment 
bear_polar_bear_facepalm

**Pull request merged**